src/cmd/clean.rs

```diff
@@ -29,7 +30,11 @@ pub fn execute(args: &ArgMatches) -> mdbook::errors::Result<()> {
         Some(dest_dir) => dest_dir.into(),
         None => book.root.join(&book.config.build.build_dir),
     };
-    fs::remove_dir_all(&dir_to_remove).chain_err(|| "Unable to remove the build directory")?;
+
+    let path_to_dir_to_remove = Path::new(&dir_to_remove);
```

Comment:
This line isn't necessary, dir_to_remove is already a Path.

Reply:
I think I was relying on my IDE too much. Checked for `.exists()` and the IDE could not find it so moved on but I was mistaken. Let me push the fix.